projects/rear_power_distribution/inc/rear_power_distribution_current_measurement.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+// Periodically reads current from load switches and exposes global storage.
+// Requires GPIO, interrupts, soft timers, ADC (in ADC_MODE_SINGLE), and I2C to be initalized.
+// I2C must be initialized on I2C_PORT_2 with the SDA and SCL addresses below.
+
+#include <stdint.h>
+#include "gpio.h"
+#include "status.h"
+
+extern const GpioAddress REAR_POWER_DISTRIBUTION_CURRENT_I2C_SDA_ADDRESS;
```

Comment:
instead of using `extern`, just define them in the `.h` file

Reply:
I get multiple definition errors defining them in .h and `extern` fixes it